content/hardware/02.hero/boards/uno-r4-wifi/tutorials/vrtc-off/vrtc-off.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Arduino UNO R4 WiFi VRTC & OFF Pins'
+description: 'Learn how to use the VRTC and OFF Pins on the UNO R4 WiFi.'
+tags:
+  - VRTC
+author: 'Jacob Hylén'
+---
+
+The Arduino UNO R4 WiFi features 2 pins that have not been seen before on UNO boards, the VRTC pin and the OFF pin. 
+
+They are used to control some of the boards functions electrically. The VRTC pin can be used to keep the onboard RTC (Real Time Clock) running even when the boards main power supply is turned off, and the OFF pin is used to turn off the board by cutting off the power.
```

Comment:
```suggestion
They are used to control some of the boards electrical functions. The VRTC pin can be used to keep the onboard RTC (Real Time Clock) running even when the boards main power supply is turned off, and the OFF pin is used to turn off the board by cutting off the power.
```

Reply:
Seemed weirdly worded, but if it is more technically correct then disregard this comment.